src/helper/remote_config/protocol/client.hpp

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <list>
+#include <string>
+
+#include "client_state.hpp"
+#include "client_tracer.hpp"
+
+namespace dds::remote_config::protocol {
+
+enum product { LIVE_DEBUGGING, ASM_DD, FEATURES };
```

Comment:
Use enum class and don't use upper-case since they will be namespaced by the name of the enum class (e.g. product::live_debugging).

Reply:
Done